### PR TITLE
adding figure captions w/ ipsum lorem

### PR DESCRIPTION
**Pull request by jordansread**
e.g.,
image

### CODE REVIEW

**Comment by jordansread** [comment]
@lindsaycarr this is a relatively small PR, but can you review and merge if 👍 ?

File: layout/templates/fishManagement.mustache [hunk 1]
```diff
@@ -11,4 +11,7 @@
 	  <div class="insetPictureContainer">
 	    <img src="images/MilleLacs_sun_boats.jpg"/>
 	  </div>
+	  <div class="captionContainer">
+	    <p class="caption">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam commodo sit amet eros quis rutrum. Morbi ipsum ligula, viverra at risus at, eleifend dignissim neque. Integer in dui tellus. Sed ut nulla odio. Mauris id eros metus.</p>
```
Comment (lindsayplatt) [inline layout/templates/fishManagement.mustache:15]: did you want to actually populate these captions? Still in Latin

Comment (jordansread) [inline layout/templates/fishManagement.mustache:15]: they are placeholders for now, but yes, eventually.

File: viz.yaml [hunk 2]
```diff
@@ -170,7 +170,7 @@ visualize:
   -
     id: futureSuitabilityFig
     location: figures/futureSuitabilityFig.svg
-    depends: processedFutureSuitability
+    depends: [processedFutureSuitability, siteTextData]
```
Comment (lindsayplatt) [inline viz.yaml:173]: I think you need to add siteTextData to the function signature for visualizeFutureSuitability?

Comment (jordansread) [inline viz.yaml:173]: hmm...it works as is, but the viz function reads in siteText internally. Agreed that it would be better to specify it. Good idea.

**Comment by lindsayplatt** [comment]
Good to go for merge, but I see some commits coming through. I don't want to prematurely merge this

**Pull request merged**